### ISSUE TITLE
Predicting the Price

### ISSUE THREAD

**Issue by Yash-Mohan-10** [reshamsai150/7.Analysis-and-Prediction-of-Electric-Vehicle-Costs-A-Machine-Learning-Based-Approach#18]
@reshamsai150 
I would like to train the model to predict the price of car
Since currently the training dataset is not given , i will use https://www.kaggle.com/datasets/fatihilhan/electric-vehicle-specifications-and-prices/data this file as my primary source of training data set 
I will be converting the german prices as per given data into INR and then based on the parameters, First normalize it, and then apply supervised learning algorithm, mostly decision trees (enhanced with random forest/XGBoost)

Tools i will use: 
1. Pandas
2. XGBoost
3. Scikit
4. Numpy

 @reshamsai150 could you please assign it to me so that i begin the implementation

**Comment by Yash-Mohan-10** [comment]
@reshamsai150 since my pr has already been accepted and merged i am closing this issue.
I would also like to know if possible where can i check the gssoc points i received for the contribution /
